src/backer/backering.py

```diff
@@ -453,31 +455,29 @@ def on_post(self, req, rep):
         if ilk not in (Ilks.icp, Ilks.rot, Ilks.ixn, Ilks.dip, Ilks.drt):
             raise falcon.HTTPBadRequest(description=f"invalid event type ({ilk})for receipting")
 
-        msg = bytearray(serder.raw)
-        msg.extend(cr.attachments.encode("utf-8"))
+        ### start ###
+        rct = self.hab.receipt(serder)
+        self.psr.parseOne(ims=bytes(rct))
```

Comment:
Hmm, we should still be calling: `self.psr.parseOne(ims=msg)` and then doing the rest of these changes after `if pre in self.hab.kevers` - otherwise we will receipt a message that might be invalid.

Reply:
(we will need to re-create or deep clone `msg` I think since `parseOne` consumes the `msg` byte stream as far as I remember, and it will be empty)

Reply:
updated
ref here: 60b40541ac1ece5dff7ef05ea8725fb0c7342700